### PR TITLE
Restrict frameworks in paket.dependencies

### PR DESCRIPTION
**Pull request by theimowski**

### CODE REVIEW

**Comment by isaacabraham** [comment]
I'm up for doing this - it reduces a lot of complexity - but I've seen strange behaviour with Paket in the past with restrictions with regards to things like updating project references and the like. Just be aware - this should be tested out fully before merging :-)

File: Content/paket.dependencies [hunk 1]
```diff
@@ -44,8 +46,8 @@ group Client
     clitool dotnet-fable ~> 2
 
 group Build
-    storage: none
     source https://api.nuget.org/v3/index.json
+    storage: none
```
Comment (theimowski) [inline Content/paket.dependencies]: question to @matthid : can fake 5 dependencies be also restricted to a certain framework? e.g. `framework: netstandard2.0` ?

**Comment by theimowski** [comment]
Yep will need to test that thoroughly, however CI shows it builds for various options - so it should be all good

File: Content/paket.dependencies (hunk 1 above)
Comment (matthid) [reply Content/paket.dependencies]: yes, in fact when using "paket-inline dependencies" `framework: netstandard2.0` will be added by default:

https://github.com/fsharp/FAKE/blob/25a04f35a00d0fe892523d992f6245edeb9e0eca/src/app/Fake.Runtime/FakeRuntime.fs#L118-L119

**Pull request merged**